src/lib/decorators.ts

```diff
@@ -265,3 +265,28 @@ export function eventOptions(options: AddEventListenerOptions) {
              // tslint:disable-next-line:no-any decorator
              any;
 }
+
+/**
+ * A property decorator that converts a class property into a getter that
+ * returns the `assignedNodes` of the given named `slot`.
+ *
+ * @ExportDecoratedItems
```

Comment:
This doesn't need to be `@ExportDecoratedItems` any more, as of https://github.com/angular/tsickle/pull/1125

Reply:
Done